apps/metadata-editor/src/app/records/all-records/all-records.component.ts

```diff
@@ -94,6 +92,27 @@ export class AllRecordsComponent {
     private cdr: ChangeDetectorRef
   ) {}
 
+  ngOnInit() {
+    this.searchFacade.init('editor')
```

Comment:
Do we really need to re-init the `searchFacade` each time we create the component? Isn't resetting the search sufficient?